tests/target-browser/target_browser_test.go

```diff
@@ -785,14 +867,106 @@ var _ = Describe("Target Browser Tests", func() {
 			})
 			// TODO update condition once API is fixed
 			It(fmt.Sprintf("Should succeed if flag %s is given 'invalid' value", flagOrderBy), func() {
-				isLast = true
 				args := []string{cmdGet, cmdBackup, flagOrderBy, "invalid"}
 				args = append(args, commonArgs...)
 				cmd := exec.Command(targetBrowserBinaryFilePath, args...)
 				_, err := cmd.CombinedOutput()
 				Expect(err).ShouldNot(HaveOccurred())
 			})
+
+			It("Should fail if path param is given 'invalidUID' value", func() {
+				isLast = true
+				args := []string{cmdGet, cmdBackup, "invalidUID"}
+				args = append(args, commonArgs...)
+				command := exec.Command(targetBrowserBinaryFilePath, args...)
+				output, err := command.CombinedOutput()
+				Expect(err).Should(HaveOccurred())
+				Expect(string(output)).Should(ContainSubstring("404 Not Found"))
+			})
+		})
+
+		Context(fmt.Sprintf("test-cases for flag %s with path value, zero, valid  and invalid value",
```

Comment:
Update desc. and variables where `backupPlan` is used instead of `backup` in this context